### DISCUSSION TITLE
Getting errors after cloned the repo

### DISCUSSION THREAD

**Discussion by moohamedsalman93** [KRTirtho/spotube#130] [Q&A] [answered]
After cloned the repo in vs code i updated all package using "flutter pub get" ,but got errors in 10th line, lib/provider/spotifyDI.dart, 

"final anonCred = getRandomElement(**spotifySecrets**);"
 
 i'm struggling to solve the issue ,plz can anyone help to resolve my issue
![spotifysecrets](https://user-images.githubusercontent.com/53552621/176899545-38b96eb9-c70b-48f8-ad9d-7706424f2db3.png)

**Comment by KRTirtho** [comment, marked as answer]
Did you read the docs (CONTRIBUTION.md) already?

**Reply by moohamedsalman93** [reply]
Yes i did 
Step 1: after cloned repo "flutter pub get"
Step 2: created secrets.json in root folder,
Step 3:"dart create-secrets.dart --local" when run this code i getting error that "create-secret.dart is not found "

**Reply by karniv00l** [reply]
There's a typo in the docs, script is under `bin` directory

**Reply by KRTirtho** [reply]
@moohamedsalman93 you've to run `dart bin/create-secrets.dart --local`

**Reply by moohamedsalman93** [reply]
thank you👍
